src/node_util.cc

```diff
@@ -174,6 +176,20 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<String> envtag = args[0]->ToString();
+  String::Utf8Value strenvtag(envtag);
+  std::string text;
+  bool isEnvSafe = false;
+  isEnvSafe = SafeGetenv(*strenvtag, &text);
+  if (isEnvSafe) {
+    args.GetReturnValue()
+    .Set(String::NewFromUtf8(isolate, text.c_str()));
+    return;
```

Comment:
Superfluous return statements.

Reply:
so no return statements are needed given the case i have 1 value to return?

Reply:
Yep.  You set the return value and then just fall through.  If you use the guard style I mentioned above, the `args.GetReturnValue().Set(...)` will also be the last statement in this function.